lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -79,6 +91,87 @@ describe('<DatePicker />', () => {
     expect(screen.queryByRole('dialog')).toBeInTheDocument();
   });
 
+  interface FormProps {
+    dateAdapter: new (...args: any) => MuiPickersAdapter<unknown>;
+    locale: any;
+    picker: React.FC<DatePickerProps>;
+    pickerProps: Partial<DatePickerProps>;
+  }
+  const Form: React.FC<FormProps> = ({ dateAdapter, locale, picker: Picker, pickerProps }) => {
+    const [date, setDate] = React.useState<unknown>(new Date('01/01/2020'));
+    return (
+      <LocalizationProvider dateAdapter={dateAdapter} locale={locale}>
+        <Picker
+          onChange={setDate}
+          renderInput={props => <TextField {...props} />}
+          value={date}
+          {...pickerProps}
+        />
+      </LocalizationProvider>
+    );
+  };
+
+  const dateAdapters = {
```

Comment:
@oliviertassinari Is this approach in alignment with how you'd like to see the adapters tested in one ci?